Cargo.toml

```diff
@@ -24,6 +24,10 @@ default = ["tokio-runtime"]
 tokio-runtime = ["tokio/macros", "tokio/net", "tokio/rt", "tokio/time", "reqwest", "serde_bytes"]
 async-std-runtime = ["async-std", "async-std/attributes", "async-std-resolver", "tokio-util/compat"]
 sync = ["async-std-runtime"]
+# The bson/u2i feature enables automatic conversion from unsigned to signed types during
+# serialization. It is preferred to use the helper functions provided in the bson::serde_helpers
+# module, which cover most use cases for this feature.
```

Comment:
I think it's worth specifically mentioning wanting to serialize/deserialize types from third-party crates (i.e. ones that you don't have access to changing the implementation of) as the intended use case for this feature. Otherwise, it might not be clear to users that it's expected in that scenario.